.circleci/config.yml

```diff
@@ -10,16 +10,27 @@ jobs:
       - image: circleci/ruby:2.5.1-node
         environment:
           BUNDLER_VERSION: 2.0.1
+          PGHOST: 127.0.0.1
+          PGUSER: omer.katzir
+          RAILS_ENV: test
 
       # Specify service dependencies here if necessary
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
+      - image: circleci/postgres:9.4
+        environment:
+          POSTGRES_USER: omer.katzir
+          POSTGRES_DB: we_eat_test
+          POSTGRES_PASSWORD: ""
```

Comment:
can this be omitted?

Reply:
user name can be blank but fields need to exist otherwise it fails